src/appengine/handlers/cron/project_setup.py

```diff
@@ -72,6 +72,10 @@
 
 PUBSUB_PLATFORMS = ['linux']
 
+MEMORY_SAFE_LANGUAGES = ['go', 'java', 'python', 'rust']
```

Comment:
Since there are fewer memory unsafe languages than memory safe ones, why don't we just use that instead?

Reply:
It is better to be explicit on which memory unsafe languages we are lowering the weight down for.